logging/WFCORE-4807-log-routing.adoc

```diff
@@ -0,0 +1,109 @@
+= Log Routing
+:author:            James R. Perkins
+:email:             jperkins@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Currently any modules that log generally log onto the system log context. In some cases, for example a deployment using
+a logging profile, the loggers may end up getting created on the wrong `LogContext`. For example if `a.war` and `b.war`
```

Comment:
What do you mean by `logging profile`? I know logging subsystem, logging modules (eg `./modules/system/layers/base/org/jboss/logging/main/module.xml`) and standalone profile, but I'm not sure what `logging profile` is.